docs/api-docs/catalog/currencies/currencies-overview.md

```diff
@@ -126,35 +126,35 @@ See the table below for a list of supported and un-supported features.
 | Name | Description |

```

Comment:
The definitions of these features don't need to restate the names in the descriptions